pkg/syncer/metadatasyncer.go

```diff
@@ -274,25 +275,6 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 			}
 		}
 
-		IsWorkloadDomainIsolationSupported = commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx,
-			common.WorkloadDomainIsolation)
-		// If the capability for WorkloadDomainIsolation feature is enabled mid-flight,
-		// this code will update the CNS-CSI FSS in vmware-system-csi namespace to true.
-		if IsWorkloadDomainIsolationSupported {
-			log.Infof("Supervisor Capability: %q is enabled", common.WorkloadDomainIsolation)
-			if !commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.WorkloadDomainIsolationFSS) {
-				log.Infof("Supervisor CNS-CSI FSS: %q is disabled", common.WorkloadDomainIsolationFSS)
-				err = commonco.ContainerOrchestratorUtility.EnableFSS(ctx, common.WorkloadDomainIsolationFSS)
```

Comment:
We are not enabling FSS, meaning we are always using Capability CR in CNS-CSI supervisor and PV-CSI Guest Cluster. Can you confirm?

Reply:
Can you also remove dead code - EnableFSS from func (c *K8sOrchestrator) EnableFSS

Also remove unnecessary RBAC rules required to update configmap.

Reply:
I had same query whether we are completely eliminating the CSI configmaps to check for WorkloadDomainIsolationFSS ??